lib/rules/color-contrast-matches.js

```diff
@@ -63,7 +63,7 @@ if (node.getAttribute('id')) {
 	}
 }
 
-if (axe.commons.text.visible(virtualNode, false, true) === '') {
+if (axe.commons.text.visibleVirtual(virtualNode, false, true) === '') {
```

Comment:
Ok, we should make that change then. It would definitely help back compat to have the older function name referenced in this rule.